checkstyle/import-control.xml

```diff
@@ -124,6 +124,7 @@
     </subpackage>
 
     <subpackage name="producer">
+      <allow pkg="org.apache.kafka.clients.consumer" />
```

Comment:
the new producer API has the `sendOffsets` method which takes `OffsetAndMetadata` in the argument. We need this line to allow `OffsetAndMetadata` to be imported.